.github/workflows/bix_docker.yml

```diff
@@ -26,10 +26,10 @@ defaults:
   run:
     shell: bash
 
-# only allow 1 run for PRs
+# only allow 1 run per commit concurrently
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ !contains(github.ref, 'master') }}
+  group: ${{ github.workflow }}-${{ github.sha }}
```

Comment:
Won't this undo the stampeding protection that we were looking for since all rebases will be on different sha's?

Reply:
oooohh. That just might. Let's try.

Reply:
It works for PRs.